src/scc_hypervisor_collector/api/uploader.py

```diff
@@ -5,7 +5,59 @@
 collected from the specified backends to the SCC using the provided
 credentials.
 """
+import json
+import logging
+import gzip
+from typing import Dict
+from importlib_metadata import version
```

Comment:
Perhaps import the version function with a more meaningful name using an `as` clause... E.g.

```
from importlib_metadata import version as get_package_version
```

This will make the calls to version below a little more "self documenting".